.gitignore

```diff
@@ -3,154 +3,21 @@
 !/build-aux
 !/build_msvc
 
-*.tar.gz
```

Comment:
I'm still not sure how I feel about removing all of these. We are going to switch a lot between the versions with and without cmake over the next two years. Is it really desirable to have all these files appear in the untracked list?

Reply:
What do you suggest?

> I'm still not sure how I feel about removing all of these. We are going to switch a lot between the versions with and without cmake over the next two years. Is it really desirable to have all these files appear in the untracked list?

From my personal experience during recent months, it isn't a problem at all.

Reply:
When I review a backport and come back again, this is how it looks like:
```
> git status
On branch master
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	Makefile
	Makefile.in
	aclocal.m4
	autom4te.cache/
... another 420 lines
```
This make using `git status` and a bunch of other git commands very annoying.